CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/TeamsAPI/Models/FileConsentCard.cs

```diff
@@ -0,0 +1,67 @@
+// <auto-generated /> Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Bot.Connector.Teams.Models
+{
+    using System.Linq;
+
+    /// <summary>
+    /// File consent card attachment.
+    /// </summary>
+    public partial class FileConsentCard
+    {
+        /// <summary>
+        /// Initializes a new instance of the FileConsentCard class.
+        /// </summary>
+        public FileConsentCard() { }
+
+        /// <summary>
+        /// Initializes a new instance of the FileConsentCard class.
+        /// </summary>
+        /// <param name="description">File description.</param>
+        /// <param name="sizeInBytes">Size of the file to be uploaded in
+        /// Bytes.</param>
+        /// <param name="acceptContext">Context sent back to the Bot if user
+        /// consented to upload. This is free flow schema and is sent back in
+        /// Value field of Activity.</param>
+        /// <param name="declineContext">Context sent back to the Bot if user
+        /// declined. This is free flow schema and is sent back in Value
+        /// field of Activity.</param>
+        public FileConsentCard(string description = default(string), long? sizeInBytes = default(long?), object acceptContext = default(object), object declineContext = default(object))
+        {
+            Description = description;
+            SizeInBytes = sizeInBytes;
+            AcceptContext = acceptContext;
+            DeclineContext = declineContext;
+        }
+
+        /// <summary>
+        /// Gets or sets file description.
+        /// </summary>
+        [Newtonsoft.Json.JsonProperty(PropertyName = "description")]
```

Comment:
Can we import the Nettonsoft.Json namespace?

Reply:
This file is auto generated. It is generated this way only by autorest.